public/javascripts/itemDetailView.js

```diff
@@ -9,8 +9,7 @@ var itemDetailView = {
 
     init: function () {
         $("#response").text("");
-        this.productID = window.args.ProductID || window.args.pageKey || window.state.ProductID;
-        this.prevPage = window.args.PreviousPage || window.state.PreviousPage;
+        this.productID = window.args.ProductID;
```

Comment:
'prevPage' no longer gets set here, but it's referenced again later on line 121. We also have 'PreviousPage' in the code which seems like the same thing with a different name.